README.md

```diff
@@ -434,7 +455,104 @@ const sdk = await init();
 sdk.applicationNavigator.openContentItem({ id: 'CONTENT_ITEM_ID...' });
 ```
 
-# Client (Supported on Content Field & Dashboard extensions)
+# Content Editor
+
+### Get Current Model
+
+Fetching will give you the current model of the content item you're viewing.
+
+```js
+const sdk = await init();
+const model = await sdk.form.getValue();
+```
+
+### Set Model
+
+Setting the model will return Error Report if there is any invalid content
+
+```js
+const sdk = await init();
+try {
+  await sdk.form.setValue({ title: "hello world", })
+} catch (errors) {
+  // retuens error report 
```

Comment:
typo above and the condition below should be in this catch block so maybe a comment isn't necessary